specification/components/stu3/schemas/endpoints/a024-retrieve-advice-and-guidance-versioned.yaml

```diff
@@ -14,6 +15,10 @@ description: |
 
   Note: e-RS only supports the retrieval of the most recent version of `CommunicationRequest`. You cannot retrieve earlier versions.
 
+  ## Important note
+ 
+  The Important notes for this endpoint are the same as for endpoint [[HYPERLINK_A024]].
+
   ## Use case
 
   See [[HYPERLINK_A005]].
```

Comment:
Should headers NHSD-End-User-Organisation-ODS, NHSD-eRS-Business-Function and NHSD-eRS-On-Behalf-Of-User-ID mention 'not allowed for application-restricted access' and 'Required for user-restricted access'? This goes for all 4 endpoints

Reply:
Raised with BA and they need to check with NHSD

Reply:
NHSD approved the changes. So we are going to include the OAS documentation on the headers